annotation_pipeline/image.py

```diff
@@ -91,7 +91,7 @@ def create_process_segment(ds_bucket, ds_segments_prefix, output_bucket, formula
     ppm = image_gen_config['ppm']
 
     def process_centr_segment(bucket, key, data_stream, ibm_cos):
-        segm_i = int(key.split("/")[-1].split(".msgpack")[0])
+        segm_i = f'{key.split("/")[-2]}/{key.split("/")[-1].split(".msgpack")[0]}'
```

Comment:
This change in `segm_i` format breaks `Pipeline.get_images`, though it's not a big deal since `Pipeline.get_images` isn't really core functionality anymore (we agreed in London that it shouldn't be part of the benchmark). 

I think this should be renamed to `segm_key` though, because the `_i` suffix means "index", and this variable now holds a complex string instead of an index.

Reply:
@LachlanStuart Thanks, in the next PR, I'll try to totally avoid using ids like this way regarding new pywren feature which extracts executions ids automatically.